tests/e2e/env/setup.sh

```diff
@@ -187,7 +190,7 @@ else
 fi
 
 echo "Installing basic auth plugin for interfacing with the API"
-cli wp plugin install https://github.com/WP-API/Basic-Auth/archive/master.zip --activate
+cli wp plugin install https://github.com/WP-API/Basic-Auth/archive/master.zip --activate --force
```

Comment:
This line had problems with installing the _Basic-Auth_ plugin when the folder was already there. Forced an overwrite if it existed.